### PR TITLE
JC-2380 SpeedUpForum

### PR DESCRIPTION
**Pull request by oatkachenko**
- Everytime when we need to get the current user, we searched it by name (case insensitive way). So indexes wasn't worked and we searched through all users. Now current user is taken from session.

### CODE REVIEW

File: jcommune-model/src/main/resources/org/jtalks/jcommune/model/entity/JCUser.hbm.xml [hunk 1]
```diff
@@ -65,7 +65,7 @@
                 type="org.joda.time.contrib.hibernate.PersistentDateTime" not-null="false"/>
       <property column="POST_COUNT" name="postCount"/>
 
-      <set name="contacts" cascade="all-delete-orphan" inverse="true">
+      <set name="contacts" cascade="all-delete-orphan" inverse="true" fetch="join">
```
Comment (ctapobep) [inline jcommune-model/src/main/resources/org/jtalks/jcommune/model/entity/JCUser.hbm.xml]: I think in most cases we don't need the contacts, do we?

Comment (oatkachenko) [inline jcommune-model/src/main/resources/org/jtalks/jcommune/model/entity/JCUser.hbm.xml]: Yes, you're right. It was needed for plugins, when created copy of JCUser. I did research and made conclusion that we don't need contacts in this scope, so I remove copying of contacts and added javadocs with explanations.

File: jcommune-service/src/main/java/org/jtalks/jcommune/service/transactional/TransactionalUserService.java [hunk 2]
```diff
@@ -82,6 +81,7 @@
     private final Base64Wrapper base64Wrapper;
     //Important, use for every password creation.
     private final EncryptionService encryptionService;
+    private SecurityContextFacade securityContextFacade;
```
Comment (ctapobep) [inline jcommune-service/src/main/java/org/jtalks/jcommune/service/transactional/TransactionalUserService.java]: `final`?

Comment (oatkachenko) [inline jcommune-service/src/main/java/org/jtalks/jcommune/service/transactional/TransactionalUserService.java]: oh it's my fault

**Pull request merged**